Add AWS NOAA Outgoing Longwave Radiation Monthly

Rub21 opened the pull request:
https://www.ncei.noaa.gov/access/metadata/landing-page/bin/iso?id=gov.noaa.ncdc:C00809

```
# python aws-noaa-outgoing-longwave-radiation-monthly/recipe.py
[########################################] | 100% Completed | 9.08 ss
<xarray.Dataset>
Dimensions:      (lat: 72, bnds: 2, lon: 144, time: 526)
Coordinates:
  * lat          (lat) float32 -88.75 -86.25 -83.75 -81.25 ... 83.75 86.25 88.75
  * lon          (lon) float32 1.25 3.75 6.25 8.75 ... 351.2 353.8 356.2 358.8
  * time         (time) datetime64[ns] 1979-01-16T12:00:00 ... 2022-10-16T12:...
Dimensions without coordinates: bnds
Data variables:
    lat_bounds   (lat, bnds) float32 dask.array<chunksize=(72, 2), meta=np.ndarray>
    lon_bounds   (lon, bnds) float32 dask.array<chunksize=(144, 2), meta=np.ndarray>
    olr          (time, lat, lon) float32 dask.array<chunksize=(526, 72, 144), meta=np.ndarray>
    time_bounds  (time, bnds) datetime64[ns] dask.array<chunksize=(526, 2), meta=np.ndarray>
Attributes: (12/47)
    Conventions:                CF-1.6, ACDD-1.3
    acknowledgment:             This project was supported in part by a grant...
    cdm_data_type:              Grid
    cdr_variable:               olr
    comment:                    none
    contributor_name:           Hai-Tien Lee
    ...                         ...
    summary:                    The product contains the 2.5-degree by 2.5-de...
    time_coverage_duration:     P526M
    time_coverage_end:          2022-11-01T:00:00:00Z
    time_coverage_resolution:   P1M
    time_coverage_start:        1979-01-01T00:00:00Z
    title:                      OLR-Monthly CDR Product Ver02Rev07

```
cc. @rbavery

cisaacstern committed:
first commit

cisaacstern committed:
correct string format for pd.date_range

cisaacstern committed:
add nitems_per_file=1 to avoid caching metadata

cisaacstern committed:
remove target_chunks, redundant w nitems_per_file

Commit:
simplify pre-commit

cisaacstern committed:
remove unnecessary spacing in recipe.py

cisaacstern committed:
Add blank line to trigger recipe runs workflow

cisaacstern committed:
Add meta.yaml with title only

cisaacstern committed:
Make meta.yaml invalid to demo validation

cisaacstern committed:
A hopefully complete & correct meta.yaml

cisaacstern committed:
Fix typo in bakery id (bot should tell me this)

cisaacstern committed:
Fix module name (bot should've told me this)

rabernat committed:
remove old recipes

rabernat committed:
rename dataset -> feestock

cisaacstern committed:
add orcid id

cisaacstern committed:
rename subdir

cisaacstern committed:
update meta.yaml

rsignell-usgs committed:
adding liveocean recipe

rsignell-usgs committed:
adding liveocean recipe

andersy005 committed:
update pre-commit and add prettier

andersy005 committed:
trigger pre-commit.ci deployment

rsignell-usgs committed:
modifying provider info for Parker MacCready

rsignell-usgs committed:
Update meta.yaml

cisaacstern committed:
downgrade recipes version for cloud environment

cisaacstern committed:
downgrade recipes version for cloud environment

cisaacstern committed:
bump back to 0.9.0 for dataflow

cisaacstern committed:
bump back to 0.9.0 for dataflow

Commit:
fix formatting

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

andersy005 committed:
fix formatting

andersy005 committed:
remove deprecated pre-commit workflow

andersy005 committed:
don't autofix PRs

jordanplanders committed:
C-iTrace materials

jordanplanders committed:
typo

jordanplanders committed:
adjusted str formatting in yaml

jordanplanders committed:
updated indent structure

jordanplanders committed:
fixed typo

jordanplanders committed:
added urls to meta.yaml

jordanplanders committed:
url formatting

rsignell-usgs committed:
Update .pre-commit-config.yaml

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

rsignell-usgs committed:
pre-commit checked

rsignell-usgs committed:
removed extra line

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

cisaacstern committed:
small change to re-trigger CI

jordanplanders committed:
correct netcdf3 type, add post processing, add tracers

jordanplanders committed:
changed title to reflect multiple tracers

jordanplanders committed:
fixed formatting errors

jordanplanders committed:
removed atmosphere variable

jordanplanders committed:
fixed post-processing to accommodate atm depth

jordanplanders committed:
Update recipes/C-iTRACE/recipe.py

change copy_input_to_local_file to True

Co-authored-by: Charles Stern <62192187+cisaacstern@users.noreply.github.com>

jordanplanders committed:
added numpy import

jordanplanders committed:
fixed yaml to reflect addition of atm c14

jordanplanders committed:
increased the number of subset_inputs

cisaacstern committed:
remove additional recipes subdirs

cisaacstern committed:
simplify recipes/README.md

cisaacstern committed:
reformat pr files with pre-commit

jordanplanders committed:
updated consolidate_zarr and subset_inputs

Co-authored-by: Charles Stern <62192187+cisaacstern@users.noreply.github.com>

brianna-corremonte committed:
updating with GPM_3IMERGHHL recipe

brianna-corremonte committed:
fix EDL issues and grouping issues with data collection compatibility with xarray

brianna-corremonte committed:
added meta.yaml removed logging and pruning to prep for aws

brianna-corremonte committed:
update yaml

brianna-corremonte committed:
adding GPM_3IMERGDL

yuvipanda committed:
Fixup meta

yuvipanda committed:
Get all

Commit:
add requirements

Commit:
pin pgf-esgf version

pangeo-forge[bot] committed:
Delete recipes/CMIP6-PMIP/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/CMIP6-PMIP/requirements.txt

pangeo-forge[bot] committed:
Delete recipes/CMIP6-PMIP/recipe.py

yuvipanda committed:
Reformat & comment code a little

brianna-corremonte committed:
updated meta.yaml for GPM_3IMERGDL

jordanplanders committed:
eVolv2k_v3 recipe

jordanplanders committed:
wrong branch

jordanplanders committed:
Revert "wrong branch"

This reverts commit 72f30667f4c6940f904354cf4355b394d8815170.

jordanplanders committed:
Revert "eVolv2k_v3 recipe"

This reverts commit a22b260196cad0d68641eba7475a37345cfe045c.

jordanplanders committed:
eVolv2k_v3 recipe

jordanplanders committed:
fixed typo in yaml file

jordanplanders committed:
removed trailing spaces

yuvipanda committed:
Remove unused recipe

yuvipanda committed:
Reformat to make pre-commit happy

yuvipanda committed:
Reformat to make pre-commit happy

Commit:
Initial commit

Commit:
Initial commit

jordanplanders committed:
double to single quotes

jordanplanders committed:
double to single quotes

jordanplanders committed:
fixed indent and added provider

jordanplanders committed:
fixed indent and added provider

Commit:
Merge pull request #191 from jbusecke/cmip-damip

CMIP6-DAMIP Feedstock

pangeo-forge[bot] committed:
Delete recipes/CMIP6-DAMIP/requirements.txt

pangeo-forge[bot] committed:
Delete recipes/CMIP6-DAMIP/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/CMIP6-DAMIP/recipe.py

pangeo-forge[bot] committed:
Delete recipes/CMIP6-DAMIP/recipe.py

pangeo-forge[bot] committed:
Delete recipes/CMIP6-DAMIP/requirements.txt

pangeo-forge[bot] committed:
Delete recipes/CMIP6-DAMIP/meta.yaml

pangeo-forge[bot] committed:
Merge pull request #192 from pangeo-forge/cleanup-1664298553

Cleanup pangeo-forge/CMIP6-DAMIP-feedstock

jordanplanders committed:
adjusted formatting

jordanplanders committed:
adjusted formatting

jordanplanders committed:
rename dict, fix troubleshoot yaml

jordanplanders committed:
rename dict, fix troubleshoot yaml

jordanplanders committed:
updated to exclude pycharm support files

jordanplanders committed:
updated to exclude pycharm support files

jordanplanders committed:
init commit

jordanplanders committed:
removed change to .gitignore about pycharm

jordanplanders committed:
removed change to .gitignore about pycharm

auraoupa committed:
removes import of unused library

yloskvi4 committed:
Update recipe.py

cisaacstern committed:
Merge pull request #190 from yuvipanda/gdl

Add GPM IMERG Late Precipitation Daily recipe

cisaacstern committed:
rever gpm merge

cisaacstern committed:
rever gpm merge

cisaacstern committed:
Merge pull request #196 from pangeo-forge/revert-gpm-merge

Revert gpm merge

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

yuvipanda committed:
Revert "rever gpm merge"

This reverts commit 96018fb53822fd90fc01ec8285e42e638a410ab1.

yuvipanda committed:
Revert "rever gpm merge"

This reverts commit 96018fb53822fd90fc01ec8285e42e638a410ab1.

yuvipanda committed:
Remove .DS_Store

yuvipanda committed:
Remove .DS_Store

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

cisaacstern committed:
Merge pull request #197 from yuvipanda/gp-2

Re-add #190 without .DS_Store

pangeo-forge[bot] committed:
Delete recipes/GPM_3IMERGDL/recipe.py

pangeo-forge[bot] committed:
Delete recipes/GPM_3IMERGDL/recipe.py

pangeo-forge[bot] committed:
Delete recipes/GPM_3IMERGDL/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/GPM_3IMERGDL/meta.yaml

pangeo-forge[bot] committed:
Merge pull request #198 from pangeo-forge/cleanup-1664392957

Cleanup pangeo-forge/GPM_3IMERGDL-feedstock

cisaacstern committed:
Update recipes/casm/recipe.py

cisaacstern committed:
move URL_FORMAT into function body

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

cisaacstern committed:
move np + crime imports into function body

cisaacstern committed:
move np + crime imports into function body

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

cisaacstern committed:
Merge pull request #188 from jordanplanders/eVolv2k_v3

eVolv2k_v3 recipe

pangeo-forge[bot] committed:
Delete recipes/eVolv2k/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/eVolv2k/recipe.py

pangeo-forge[bot] committed:
Delete recipes/eVolv2k/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/eVolv2k/recipe.py

pangeo-forge[bot] committed:
Merge pull request #199 from pangeo-forge/cleanup-1664408400

Cleanup pangeo-forge/eVolv2k-feedstock

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

andersy005 committed:
Merge pull request #66 from cisaacstern/soda3.4.2-ice

pangeo-forge[bot] committed:
Delete recipes/soda-342/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/soda-342/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/soda-342/soda342_ice.py

pangeo-forge[bot] committed:
Delete recipes/soda-342/soda342_ice.py

pangeo-forge[bot] committed:
Merge pull request #200 from pangeo-forge/cleanup-1664411499

Cleanup pangeo-forge/soda-342-feedstock

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

andersy005 committed:
update recipe metadata

andersy005 committed:
Merge remote-tracking branch 'origin' into EOBS_meta_recipe

* origin: (93 commits)
  Delete recipes/soda-342/soda342_ice.py
  Delete recipes/soda-342/meta.yaml
  [pre-commit.ci] auto fixes from pre-commit.com hooks
  Delete recipes/eVolv2k/recipe.py
  Delete recipes/eVolv2k/meta.yaml
  [pre-commit.ci] auto fixes from pre-commit.com hooks
  move np + crime imports into function body
  Delete recipes/GPM_3IMERGDL/recipe.py
  Delete recipes/GPM_3IMERGDL/meta.yaml
  [pre-commit.ci] auto fixes from pre-commit.com hooks
  Remove .DS_Store
  Revert "rever gpm merge"
  rever gpm merge
  Reformat to make pre-commit happy
  removed change to .gitignore about pycharm
  updated to exclude pycharm support files
  rename dict, fix troubleshoot yaml
  adjusted formatting
  Delete recipes/CMIP6-DAMIP/requirements.txt
  Delete recipes/CMIP6-DAMIP/recipe.py
  ...

andersy005 committed:
Merge pull request #194 from yloskvi4/master

pangeo-forge[bot] committed:
Delete recipes/casm/recipe.py

pangeo-forge[bot] committed:
Delete recipes/casm/meta.yaml

pangeo-forge[bot] committed:
Merge pull request #201 from pangeo-forge/cleanup-1664415522

Cleanup pangeo-forge/casm-feedstock

jordanplanders committed:
tiny typo

jordanplanders committed:
fixed renaming scheme

derekocallaghan committed:
As suggested, I've moved the `metpy` import into the body of `ics_wind_speed_direction()`

norlandrhagen committed:
Merge branch 'pangeo-forge:master' into EOBS_meta_recipe

norlandrhagen committed:
fixed recipe ID's

derekocallaghan committed:
All imports required by `ics_wind_speed_direction()` have been moved into the function body.

andersy005 committed:
Update license

andersy005 committed:
Merge pull request #193 from jordanplanders/LMRv2p1_MCruns_ensemble_gridded

pangeo-forge[bot] committed:
Delete recipes/LMRv2p1_MCruns_ensemble_gridded/recipe.py

pangeo-forge[bot] committed:
Delete recipes/LMRv2p1_MCruns_ensemble_gridded/meta.yaml

pangeo-forge[bot] committed:
Merge pull request #202 from pangeo-forge/cleanup-1664517581

Cleanup pangeo-forge/LMRv2p1_MCruns_ensemble_gridded-feedstock

pre-commit-ci[bot] committed:
[pre-commit.ci] pre-commit autoupdate

updates:
- [github.com/asottile/pyupgrade: v2.37.3 → v2.38.2](https://github.com/asottile/pyupgrade/compare/v2.37.3...v2.38.2)
- [github.com/psf/black: 22.6.0 → 22.8.0](https://github.com/psf/black/compare/22.6.0...22.8.0)
- [github.com/keewis/blackdoc: v0.3.5 → v0.3.7](https://github.com/keewis/blackdoc/compare/v0.3.5...v0.3.7)
- [github.com/pre-commit/mirrors-prettier: v2.7.1 → v3.0.0-alpha.0](https://github.com/pre-commit/mirrors-prettier/compare/v2.7.1...v3.0.0-alpha.0)

cisaacstern committed:
Merge pull request #203 from pangeo-forge/pre-commit-ci-update-config

[pre-commit.ci] pre-commit autoupdate

derekocallaghan committed:
Added datetime import to ics_wind_speed_direction()

andersy005 committed:
Merge pull request #189 from auraoupa/master

pangeo-forge[bot] committed:
Delete recipes/eNATL60/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/eNATL60/recipe.py

pangeo-forge[bot] committed:
Merge pull request #210 from pangeo-forge/cleanup-1666288761

Cleanup pangeo-forge/eNATL60-feedstock

andersy005 committed:
Merge main into liveocean

andersy005 committed:
reset setup.cfg

norlandrhagen committed:
moved globally defined vars into functions to satisfy beam

norlandrhagen committed:
Merge pull request #169 from norlandrhagen/EOBS_meta_recipe

consolidated three EOBS recipe in meta recipe

pangeo-forge[bot] committed:
Delete recipes/EOBS/recipe.py

pangeo-forge[bot] committed:
Delete recipes/EOBS/meta.yaml

pangeo-forge[bot] committed:
Merge pull request #211 from pangeo-forge/cleanup-1666380749

Cleanup pangeo-forge/EOBS-feedstock

raf-antwerpen committed:
add recipe and meta (first draft)

raf-antwerpen committed:
add recipe and meta

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

raf-antwerpen committed:
Update recipes/mar_greenland/meta.yaml

Co-authored-by: Julius Busecke <julius@ldeo.columbia.edu>

raf-antwerpen committed:
Update recipes/mar_greenland/meta.yaml

Co-authored-by: Anderson Banihirwe <axbanihirwe@gmail.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

raf-antwerpen committed:
Update recipes/mar_greenland/recipe.py

Co-authored-by: Anderson Banihirwe <axbanihirwe@gmail.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

raf-antwerpen committed:
Update recipes/mar_greenland/recipe.py

Co-authored-by: Anderson Banihirwe <axbanihirwe@gmail.com>

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

andersy005 committed:
Merge pull request #145 from eooffshore/eooffshore_ics_ccmp_v02_1_nrt_wind

pangeo-forge[bot] committed:
Delete recipes/eooffshore_ics_ccmp_v02_1_nrt_wind/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/eooffshore_ics_ccmp_v02_1_nrt_wind/recipe.py

pangeo-forge[bot] committed:
Merge pull request #214 from pangeo-forge/cleanup-1667230523

Cleanup pangeo-forge/eooffshore_ics_ccmp_v02_1_nrt_wind-feedstock

andersy005 committed:
subset inputs to avoid memory issues

raf-antwerpen committed:
updated url in meta file

raf-antwerpen committed:
Merge branch 'mar-greenland' of https://github.com/raf-antwerpen/staged-recipes into mar-greenland

raf-antwerpen committed:
Update meta.yaml

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

raf-antwerpen committed:
Apply suggestions from code review

Co-authored-by: Anderson Banihirwe <axbanihirwe@gmail.com>

raf-antwerpen committed:
Apply suggestions from code review

Co-authored-by: Anderson Banihirwe <axbanihirwe@gmail.com>

raf-antwerpen committed:
Update meta.yaml

raf-antwerpen committed:
Update meta.yaml

andersy005 committed:
Trigger CI

andersy005 committed:
Reset .gitignore so as to fix the GitHub App

andersy005 committed:
Merge pull request #212 from raf-antwerpen/mar-greenland

pangeo-forge[bot] committed:
Delete recipes/mar_greenland/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/mar_greenland/recipe.py

pangeo-forge[bot] committed:
Merge pull request #216 from pangeo-forge/cleanup-1667494733

Cleanup pangeo-forge/mar_greenland-feedstock

pre-commit-ci[bot] committed:
[pre-commit.ci] pre-commit autoupdate

updates:
- [github.com/asottile/pyupgrade: v2.38.2 → v3.2.0](https://github.com/asottile/pyupgrade/compare/v2.38.2...v3.2.0)
- [github.com/psf/black: 22.8.0 → 22.10.0](https://github.com/psf/black/compare/22.8.0...22.10.0)
- [github.com/keewis/blackdoc: v0.3.7 → v0.3.8](https://github.com/keewis/blackdoc/compare/v0.3.7...v0.3.8)
- [github.com/pre-commit/mirrors-prettier: v3.0.0-alpha.0 → v3.0.0-alpha.4](https://github.com/pre-commit/mirrors-prettier/compare/v3.0.0-alpha.0...v3.0.0-alpha.4)

andersy005 committed:
Merge pull request #218 from pangeo-forge/pre-commit-ci-update-config

kathrynberger committed:
add file pattern

kathrynberger committed:
replace with ref_hdf_zarr recipe and add meta yaml

kathrynberger committed:
additions to meta file

kathrynberger committed:
cleaned up recipe

kathrynberger committed:
Update recipes/aws-noaa-whoi/meta.yaml

revise with active bakery

Co-authored-by: Charles Stern <62192187+cisaacstern@users.noreply.github.com>

kathrynberger committed:
revise with deterministic file pattern

kathrynberger committed:
add s3 prefix and allow for future years

rbavery committed:
aot daily

rbavery committed:
remove req file

cisaacstern committed:
Merge pull request #179 from rsignell-usgs/liveocean

Liveocean

pangeo-forge[bot] committed:
Delete recipes/liveocean/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/liveocean/recipe.py

pangeo-forge[bot] committed:
Merge pull request #225 from pangeo-forge/cleanup-1669912584

Cleanup pangeo-forge/liveocean-feedstock

Commit:
Add AWS NOAA Outgoing Longwave Radiation Monthly

cisaacstern committed:
Merge pull request #221 from developmentseed/noaa-whoi

AWS NOAA WHOI

pangeo-forge[bot] committed:
Delete recipes/aws-noaa-whoi/recipe.py

pangeo-forge[bot] committed:
Delete recipes/aws-noaa-whoi/meta.yaml

pangeo-forge[bot] committed:
Merge pull request #229 from pangeo-forge/cleanup-1670964512

Cleanup pangeo-forge/aws-noaa-whoi-feedstock

rbavery committed:
import pandas in format func

rbavery committed:
import pandas top level

rbavery committed:
copy var to func

sharkinsspatial committed:
Fix pre-commit errors.

sharkinsspatial committed:
Merge pull request #232 from developmentseed/aws-noaa-oisst

AWS NOAA Optimum Interpolation SST

pangeo-forge[bot] committed:
Delete recipes/aws-noaa-oisst/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/aws-noaa-oisst/recipe.py

pangeo-forge[bot] committed:
Merge pull request #234 from pangeo-forge/cleanup-1671208390

Cleanup pangeo-forge/aws-noaa-oisst-feedstock

rbavery committed:
hgg basic

rbavery committed:
remove empty lines

sharkinsspatial committed:
Merge pull request #235 from developmentseed/noaa-cdr-isscp-hgg-basic

NOAA CDR ISSCP HGG Basic Version

rbavery committed:
hgh recipe

rbavery commented on recipes/aws-noaa-outgoing-longwave-radiation-monthly/meta.yaml:
```diff
@@ -0,0 +1,22 @@
+title: 'AWS NOAA OUTGOING LONGWAVE RADIATION MONTHLY'
+description: 'Analysis-ready datasets derived from NOAA OUTGOING LONGWAVE RADIATION MONTHLY'
```
can you fill in descriptions from the documentation webpages instead of using the title in this field?

https://www.ncei.noaa.gov/products/climate-data-records/outgoing-longwave-radiation-daily

Commit:
Update description for outgoing-longwave-radiation-monthly

sharkinsspatial committed:
Merge pull request #237 from developmentseed/noaa-cdr-isscp-hgh-basic

NOAA CDR ISCCP HGH Basic

pangeo-forge[bot] committed:
Delete recipes/noaa-atmosphere-climate-cloud-properties-isccp-hgh-basic/meta.yaml

pangeo-forge[bot] committed:
Delete recipes/noaa-atmosphere-climate-cloud-properties-isccp-hgh-basic/recipe.py

pangeo-forge[bot] committed:
Merge pull request #241 from pangeo-forge/cleanup-1671576426

Cleanup pangeo-forge/noaa-atmosphere-climate-cloud-properties-isccp-hgh-basic-feedstock

pre-commit-ci[bot] committed:
[pre-commit.ci] pre-commit autoupdate

updates:
- [github.com/pre-commit/pre-commit-hooks: v4.3.0 → v4.4.0](https://github.com/pre-commit/pre-commit-hooks/compare/v4.3.0...v4.4.0)
- [github.com/asottile/pyupgrade: v3.2.0 → v3.3.1](https://github.com/asottile/pyupgrade/compare/v3.2.0...v3.3.1)
- [github.com/psf/black: 22.10.0 → 22.12.0](https://github.com/psf/black/compare/22.10.0...22.12.0)
- [github.com/PyCQA/flake8: 5.0.4 → 6.0.0](https://github.com/PyCQA/flake8/compare/5.0.4...6.0.0)
- [github.com/PyCQA/isort: 5.10.1 → 5.11.4](https://github.com/PyCQA/isort/compare/5.10.1...5.11.4)

andersy005 committed:
Merge pull request #227 from pangeo-forge/pre-commit-ci-update-config

cisaacstern committed:
Delete recipes/noaa-atmosphere-climate-cloud-properties-isccp-hgg-basic directory

cisaacstern committed:
Merge pull request #244 from pangeo-forge/remove-hgg

Remove un-cleaned-up subdir from recipes directory

Rub21 commented:
Adding testing code and creating plot to check the result from `recipe_pruned`

```py

from dask.diagnostics import ProgressBar
import intake

recipe_pruned = recipe.copy_pruned()
delayed = recipe_pruned.to_dask()
with ProgressBar():
    delayed.compute()
cat_url = f"{recipe_pruned.target}/reference.yaml"
cat = intake.open_catalog(cat_url)
ds = cat.data.to_dask()

ds.olr.mean(dim="time").plot(size=6)

```
<img width="691" alt="image" src="https://user-images.githubusercontent.com/1152236/211672936-7be549e6-5522-4856-a2c0-7b3b165d4677.png">

rbavery committed:
change time n_items to 526 since there are 526 time steps per file

rbavery reviewed (commented):
approved but needs to be merged to master
